content/about/index.md

```diff
@@ -0,0 +1,122 @@
++++
+fragment = "content"
+weight = 10
+background = "light"
+title = "About Geostyler"
++++
+
+Geostyler is about styling maps. Maybe you are a developer looking for a set of UI components 
+to build custom and flexible styling widgets for your geospatial application. Maybe you 
+are a cartographer trying to get the style you worked so hard on in your favorite (Q)GIS into the 
+web. Or maybe you are a project lead worrying about migrating all your styles to that new fancy 
+rendering engine everyone (and particular your boss) is talking about. The Geostyler ecosystem is 
+here to help. 
+
+###  React Library
```

Comment:
Maybe we should add a sentence that emphasizes that the UI can be used with any supported styling format, as this is the key selling point and main advantage in comparison to other UIs